cli/config/configfile/file.go

```diff
@@ -36,6 +36,10 @@ type ConfigFile struct {
 	Filename             string                      `json:"-"` // Note: for internal use only
 	ServiceInspectFormat string                      `json:"serviceInspectFormat,omitempty"`
 	ServicesFormat       string                      `json:"servicesFormat,omitempty"`
+	HTTPProxy            string                      `json:"httpProxy,omitempty"`
+	HTTPSProxy           string                      `json:"httpsProxy,omitempty"`
+	NoProxy              string                      `json:"noProxy,omitempty"`
+	FTPProxy             string                      `json:"ftpProxy,omitempty"`
```

Comment:
I'm not sure what the convention is here. Personally I'd prefer `http-proxy`, `https-proxy` etc.. but from what I saw in the code it seems that `camelCase` is the done thing. Happy to change to whatever.